docs/source/upgrade.md

```diff
@@ -2,6 +2,24 @@
 
 This pages describes Scylla Operator upgrade procedures.
 
+## Upgrade via Helm
+Replace `<release_name>` with the name of your Helm release for Scylla Operator and 
+replace `<version>` with the version number you want to install:
+1. Make sure Helm chart repository is up-to-date:
+   ```
+   helm repo add scylla-operator https://storage.googleapis.com/scylla-operator-charts/stable
+   helm repo update
+   ```
+2. Update CRD resources
+   ```
+   tmpdir=$( mktemp -d ) && helm pull scylla-operator/scylla-operator --version <version> --untar --untardir "${tmpdir}" && find "${tmpdir}"/scylla-operator/crds/ -name '*.yaml' -printf '-f=%p ' | xargs kubectl apply
```

Comment:
`--server-side` (is important for more actors, maybe a note it's fine without it for old versions that don't have it)

Reply:
good point, done

Reply:
done